### ISSUE TITLE
Release 0.3.7.post1

### ISSUE THREAD

**Issue by garrettr** [freedomofpress/securedrop#1335]
As loathe as I am to have 3 dots in a version number, ever, I think we need to another release to fix the issues in the documentation identified by #1322 and #1325.

Since fixing these minor errors will not affect the distributed software, I think it is appropriate to issue a "post release", according to the conventions of [PEP 440](https://www.python.org/dev/peps/pep-0440/#post-releases). 

This release will not change any code or configuration for a production deployment; as a result, it is not necessary to update the version number displayed on the Source Interface (`securedrop/version.py`), and it is not necessary to release new packages on the Debian package server. Thus, the versions displayed there will remain "0.3.7" for 0.3.7.post1.

0.3.7.post1 will only fix documentation, and the related `update_version.sh` script. Once it is tagged and released, ReadTheDocs' **stable** documentation will automatically be updated to reflect it (because 0.3.7.post1 > 0.3.7, see the [ReadTheDocs docs on versioning](https://read-the-docs.readthedocs.io/en/latest/versions.html)). I double-checked this by pushing a dummy 0.3.7.post1 tag and confirmed that ReadTheDocs considers 0.3.7.post1 > 0.3.7 for the purposes of determining the **stable** version.

The documentation itself will continue to refer to 0.3.7, in order to avoid a confusing discrepancy between the version number displayed in the documentation and the version number displayed on the Source Interface.

We should write a short blog post when this is released, to explain the motivation behind it and avoid confusion. Since this change will only affect the git repo, it will mostly be invisible to end users.

What say you, @freedomofpress/securedrop-maintainers?

**Comment by fowlslegs** [comment]
Do you really think this will really cause a lot of confusion for people and even if it does, do you think they would intuitively know to go to our blog to figure it out? It wouldn't hurt to write a blog post if someone really wants to, but either way, there does need to be a post release. The one thing to keep in mind is that currently 0.3.8 is branched of 0.3.7, so we'll need to cherry pick the commits from 0.3.7.post1 into 0.3.8. Since it's just docs there should be no merging conflicts.

**Comment by garrettr** [comment]
> The one thing to keep in mind is that currently 0.3.8 is branched of 0.3.7, so we'll need to cherry pick the commits from 0.3.7.post1 into 0.3.8. Since it's just docs there should be no merging conflicts.

There aren't any new commits on release/0.3.8 yet, so a merge would just be a fast-forward. I would just merge (or rebase) 0.3.7.post1 onto 0.3.8 to keep the history linear.

**Comment by garrettr** [comment]
> Do you really think this will really cause a lot of confusion for people and even if it does, do you think they would intuitively know to go to our blog to figure it out? It wouldn't hurt to write a blog post if someone really wants to,

Yeah, on second thought, a blog post is probably unnecessary. I think this public discussion on GitHub is sufficient for the purposes of transparency.

**Comment by fowlslegs** [comment]
> I would just merge (or rebase) 0.3.7.post1 onto 0.3.8 to keep the history linear.

That makes more sense.

**Comment by garrettr** [comment]
I did a quick internal poll to see if there's anything else that should go into 0.3.7.post1, and everyone agreed that it is ready to go. All outstanding PR's have been merged into `release/0.3.7.post1`.

**Comment by garrettr** [comment]
A new tag, [0.3.7.post1](https://github.com/freedomofpress/securedrop/tree/0.3.7.post1), has been pushed to mark this release. I have verified that ReadTheDocs has rebuilt the [stable documentation](https://docs.securedrop.org/en/stable) based on the 0.3.7.post1 tag, which fixes the issues in the documentation for end users.

In addition, I have changed ReadTheDocs' settings so the `stable` documentation is the default version of the documentation shown. This is more appropriate since the built documentation is intended for end users, who should be reading the stable documentation. The `latest` documentation is still being built from the develop branch, but that should only be used by SecureDrop developers.
